src/db/collections/ledger_update.rs

```diff
@@ -140,12 +143,14 @@ impl MongoDb {
             let is_spent = metadata.spent.is_some();
 
             // Ledger updates
-            for owner in output.owning_addresses() {
+            for (address, unlock_condition) in output.owning_addresses() {
```

Comment:
`unlock_condition_desc` is probably clearer to not confuse it with the "real" `UnlockCondition` type.